Created a new footer based on the CSS

plandsman11 opened the pull request:
I created a new Footer based on the Figma. It looks like the following and resizes based on percents. The only thing that could need to be changed is that at the moment the footer is in a rectangular background. The CSS has the background being wavy, not sure how to achieve this without just taking a screenshot. 

<img width="1538" alt="Screen Shot 2021-04-19 at 5 31 41 PM" src="https://user-images.githubusercontent.com/59515892/115319645-1c981680-a135-11eb-9eee-dc8e5869ca0c.png">

plandsman11 committed:
Created a new footer based on the CSS

plandsman11 commented:
Also, we need to add functionality to the social media logos. Do we want to include these?

bglossner commented on frontend/src/components/Footer/Footer.js:
```diff
@@ -0,0 +1,74 @@
+                </div>
+            </div>
+            <div className="column"> 
+                <p className="middle-info"> &copy; 2019 Happy Hats for Kids in Hospitals</p>
+                <p className="middle-info"> Happy hats For Kids In Hospitals is a California 501(C)(3) Nonprofit Organization</p>
+                <p className="middle-info"> Founded in 1991 by Sheri Schrier</p>
```
What's the middle-info class?

plandsman11 replied:
That is just for formatting the middle column text.

plandsman11 committed:
Removed the social media logos and replaced it with h4i statement

plandsman11 commented:
Removed the social media logos and replaced it with h4i statement
<img width="1012" alt="Screen Shot 2021-04-19 at 6 25 02 PM" src="https://user-images.githubusercontent.com/59515892/115323260-a5ff1700-a13c-11eb-9917-7b5035dd769a.png">
